README.md

```diff
@@ -21,20 +21,20 @@ Hit `install` button.
 #### AssignMisused
 Detects assignment and comparison operators in one statement.
 ```php
-do{
+do {
  //...
-}while( false == $user->getName() = self::ADMIN_USER );
+} while( false == $user->getName() = self::ADMIN_USER );
```

Comment:
This statement cannot be executed in PHP.
Cannot find a proper alternative example, but it should be updated once.

Reply:
I think `while (false == $admin = $user->getName() )` should be good here, WDYT?

Reply:
Falsable username is not clear to the developer for an understanding of the issue.
Also, I cannot find a case for assigning in do-while to reuse it later. 

An adapted current example may be like this.
```php
$adminReceived = false;
do {
  // false
while (false === $adminReceived = $user->isAdmin());
```

Reply:
In most cases, variables are changed in a `do` body.
Your example is good. 

BTW example from the Symfony framework:
```php
while (false !== $current = ldap_next_entry($con, $current)) {
  yield $this->getSingleEntry($con, $current);
}
```